src/Cake.Frosting/Cake.Frosting.csproj

```diff
@@ -5,6 +5,7 @@
     <Copyright>Copyright (c) .NET Foundation and contributors</Copyright>
     <Authors>Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström and contributors</Authors>
     <PackageTags>Cake;Build;Build automation</PackageTags>
+    <PackageIcon>cake-medium.png</PackageIcon>
     <PackageIconUrl>https://raw.githubusercontent.com/cake-build/graphics/master/png/cake-medium.png</PackageIconUrl>
```

Comment:
PackageIconUrl and PackageIcon shouldn't be combined it's either or.

Reply:
According to the tip in https://docs.microsoft.com/en-us/nuget/reference/nuspec which states `You can specify both icon and iconUrl to maintain backward compatibility with sources that do not support icon.`, I left the Url in. But it's fine for me to remove it.

Reply:
Think we need to update the SDK for it to work properly
Icon is added but `PackageIcon` ignored dotnet by pack so it doesn't end up in the final nuspec

Reply:
Raised issue #73 for that.

Reply:
Makes probably sense to anyway go to the latest LTS.

Reply:
Agreed.